docs/migration.md

```diff
@@ -0,0 +1,15 @@
+---
+layout: index
+title: Contributing
+navigation_weight: 7
+---
+# Migration from sw-framework
+
+[Intro]
+
+## Command-Line Interface (sw-precache)
+
+Just like `sw-precache`, Workbox allows you generate a service worker using the command line using [`sw-cli`](sw-cli). There are a few differences between the two.
+
+* Information that was previously passed as command line arguments is now gathered by a wizard that provides the option of saving your choices to a config file (`sw-cli-config.json`).
```

Comment:
Jeff's correct here. As much as possible we're trying to maintain some level of parity in how sw-cli is configured compared to sw-precache and they both do support CLI flags and a config.